sdk/typescript/entrypoint/register.ts

```diff
@@ -5,7 +5,7 @@ import {
   ModuleID,
   TypeDef,
   TypeDefKind,
-} from "../api/client.gen"
+} from "../api/client.gen.js"
```

Comment:
It doesn't compile if I omit this :(

Reply:
Sorry, I'm not suggesting a change here 🙂 Just that it reminded me of Node's creator regretting having support for **not putting** the extension (youtube link above), like it was before. That was for `require()` though, because it makes node go look into the various files that could match this (is it .js, .ts, etc...?).